src/crimson/os/seastore/btree/btree_range_pin.h

```diff
@@ -12,6 +12,8 @@
 
 namespace crimson::os::seastore {
 
+constexpr uint16_t MAX_FIXEDKVBTREE_DEPTH = 8;
```

Comment:
Follow up the comment https://github.com/ceph/ceph/pull/47878#discussion_r961309631

We should be aware and fail if the tree depth is above the MAX during root split: https://github.com/ceph/ceph/blob/2c21cc1b0bcc537d558a04e0756e1ff8a8a55744/src/crimson/os/seastore/btree/fixed_kv_btree.h#L1408-L1422

Reply:
done.